### PR TITLE
ADD CARD PAYMENT COMPONENT #43

### PR DESCRIPTION
**Pull request by abdulmaleekadams**
It has a slider to toggle the dark mode and light mode
image
image

### CODE REVIEW

File: src/routes/card-payment/payment.css [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+}
+
+.cardPaymentHeader {
+  font-weight: 800;
+  line-height: 2.1875rem;
+  font-size: 1.75rem;
+}
+
+.darkMode {
+  /* background-color: #100D1F; */
```
Comment (aftabrehan) [inline src/routes/card-payment/payment.css:141]: Please remove commented code.

**Comment by abdulmaleekadams** [comment]
image
image

Screenshots of the components

File: src/index.css [hunk 2]
```diff
@@ -12,15 +12,13 @@ body {
...
-
-input {
+input{
   outline: none;
   border: none;
 }
 
+a{
+=======
 a {
```
Comment (aftabrehan) [inline src/index.css:22]: This conflict should be resolved.
```suggestion
a {
```

**Review by aftabrehan — CHANGES_REQUESTED** [review]
Excellent PR, @abdulmaleekadams 💪 🚀 💯 

Just needs minor fixes and then it's going 🚀

**Review by aftabrehan — APPROVED** [review]
LGTM @abdulmaleekadams 👍💯💪

File: src/routes/card-payment/payment.css (hunk 1 above)
Comment (abdulmaleekadams) [reply src/routes/card-payment/payment.css]: Donee

**Comment by abdulmaleekadams** [comment]
@aftabrehan  Thank you

**Pull request merged**